core/src/templates/components/alert/alert.twig

```diff
@@ -19,13 +19,13 @@
  * - alert_body: Text details of the alert message.
  */
 #}
-<div {{ attributes }} class="su-alert {{ modifier_class }}" >
-  <div class="su-alert__body">
+<article {{ attributes }} class="su-alert {{ modifier_class }}" role="alert" >
+  <section class="su-alert__body">
     {% if alert_header is not empty %}
     <h3 class="su-alert__heading">{{ alert_header }}</h3>
     {% endif %}
     {% if alert_text is not empty %}
     <p class="su-alert__text">{{ alert_text }}</p>
     {% endif %}
-  </div>
-</div>
+  </section>
+</article>
```

Comment:
@yvonnetangsu, 
I'd love your $0.02 on this.

Reply:
Will do after next week :D Rushing to the airport!

Reply:
@cjwest what would make this an article?

Reply:
Thank you for asking about the `<article>` element. It may not be appropriate. What I was thinking was that an alert is a message is generally independent of the text on a page. That is:

> The HTML `<article>` element represents a self-contained composition in a document, page, application, or site, which is intended to be independently distributable or reusable (e.g., in syndication).

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article

The alert is
1. Self contained
2. reusable

It is not intended for syndication, so that might be the show stopper. In that case, I wonder if a `<div>` would probably be more appropriate. 

I have not found any examples of using HTML5 for this kind of use case. Maybe I'm searching the wrong terms?

I welcome your thoughts.

Reply:
Thanks for the explanation, I think this all makes sense. Good to have it documented here as well.

Reply:
Hmmm....I prefer `<section>` over `<article>` myself, since an alert sometimes only make sense when it's shown together with some other elements on a page (eg, alert of a successful submission) so it's not quite independently distributable in that case 😃

Reply:
Perhaps `<div>` from https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section

Reply:
I don't think `<section>` would be appropriate:
> A rule of thumb is that a section should logically appear in the outline of a document.

I think either `<div>` or `<article>` would be appropriate. I'm still leaning towards `<article>`. With regard to the alert being related to a successful submission, I think the submission is as not _necessarily_ related to a page. That is, a submission is with regard to an _action_, not really the content. 

I welcome your thoughts, @yvonnetangsu and @sherakama.